pytest_fixtures/component/taxonomy.py

```diff
@@ -90,6 +90,19 @@ def module_sca_manifest_org(module_org, module_sca_manifest, module_target_sat):
     return module_org
 
 
+@pytest.fixture(scope='module')
+def module_extra_rhel_entitlement_manifest_org(
+    module_target_sat,
+    module_org,
+    module_extra_rhel_entitlement_manifest,
+):
+    """Creates an organization and uploads an entitlement mode manifest generated by manifester and
+    containing more RHEL entitlements than the default entitlement manifest"""
+    module_target_sat.upload_manifest(module_org.id, module_extra_rhel_entitlement_manifest.content)
+    module_org.sca_disable()
```

Comment:
FYI, BZ:2159974 is on_qa, so can we just disable SCA for org first, then upload manifest?

Reply:
Sure, that should be fine. I'll make the change.

Reply:
not seeing this updated?